lib/Conch/Validation/SasSsdNum.pm

```diff
@@ -19,24 +19,15 @@ sub validate {
 	my @disks_with_drive_type =
 		grep { $_->{drive_type} } ( values $data->{disks}->%* );
 
-	my $ssd_num = grep {
+	my $sas_ssd_count = grep {
 		fc( $_->{drive_type} ) eq fc('SAS_SSD')
-			|| fc( $_->{drive_type} ) eq fc('SATA_SSD')
 	} @disks_with_drive_type;
 
-	my $ssd_want = $hw_profile->ssd_num || 0;
-
-	# Joyent-Compute-Platform-3302 special case.  HCs can have 8 or 16 SSD and
-	# there's no other identifier. Here, we want to avoid missing
-	# failed/missing disks, so we jump through a couple extra hoops.
-	if ( $self->hardware_product_name eq "Joyent-Compute-Platform-3302" ) {
-		if ( $ssd_num <= 8 ) { $ssd_want = 8; }
-		if ( $ssd_num > 8 )  { $ssd_want = 16; }
-	}
```

Comment:
Won't this kill our ability to revalidate these devices? Does that matter?

Reply:
This test has been moved to where[ it should be](https://github.com/joyent/conch/pull/590/files#diff-be4ef5f6bab66514b39768047effbd31R25) and the testing of reports from JCP-3302s should not be impacted.